get_empirical_dist.R

```diff
@@ -99,6 +106,8 @@ if(! is.na(opt$exclusions)){
 
 reference_labs_df = read.csv(opt$input_ref_file, sep="\t", stringsAsFactors=FALSE, comment.char = "#")
 reference_labs = reference_labs_df[, opt$label_column_ref]
+sample_prop = opt$sample_prop
+sample_size = round(length(reference_labs)*sample_prop, 0)
```

Comment:
I don't think you need this line, you could just specify the proportion in the `sample` function. Have a look at it's docs, I think it should be explained there

Reply:
`sample` function takes an integer for sample size, so I'm not quite sure about how to avoid this line...

Reply:
ah, then I'm confusing it with Python syntax, sorry